apps/odyssey-stats/src/styles/wp-admin.scss

```diff
@@ -16,6 +16,12 @@
 	font-size: 13px; /* stylelint-disable-line declaration-property-unit-allowed-list */
 	margin-bottom: 2px;
 
+	// Avoid overriding hidden notices.
+	&[aria-hidden="true"],
```

Comment:
should we rename our `.notice` class if this happens again?

Reply:
This idea might be worth considering if we encounter a similar regression in the future.

Reply:
> should we rename our .notice class

This way seems to be the most fundamental to resolving those issues. However, some [CSS selector overriding](https://github.com/search?q=repo%3AAutomattic%2Fwp-calypso+.notice+language%3ASCSS&type=code&l=SCSS&p=1) should be considered when refactoring the `Notice` component.